DeadCrumbs/app/src/main/java/dead/crumbs/ui/RSSIProximity.kt

```diff
@@ -0,0 +1,42 @@
+package dead.crumbs.ui
+
+import dead.crumbs.data.BluetoothRSSI
+import kotlin.math.exp
+
+private val NUMBER_MEASUREMENTS = 5; //Number of measurements stored for each mac address - used for average
```

Comment:
Out of curiosity, is there a specific reason they are in global scope (scope of the file) and not the class?

Reply:
This is done because it looks nice